benchmarks/conftest.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+import pytest
+
+from superphot_plus.mlp import MLP
+
+BENCHMARKS_DIR = os.path.dirname(__file__)
+
+DATA_DIR_NAME = "data"
+
+# pylint: disable=missing-function-docstring, redefined-outer-name
+
+
+@pytest.fixture
+def benchmarks_data_dir():
+    return os.path.join(BENCHMARKS_DIR, DATA_DIR_NAME)
+
+
+@pytest.fixture
+def classifier(benchmarks_data_dir):
+    filename = os.path.join(benchmarks_data_dir, "superphot-model-ZTF23aagkgnz.pt")
```

Comment:
Should it be consistent with `single_ztf_id()`? I actually don't understand this prefix in a model name, how can a classification model been trained on a single source?..

Reply:
You're right, the model wasn't trained on a single source. I am not sure exactly why but the current implementation of MLP saves the pretained model with the name of the first test source.